doc/tutorials/bare_flame_game.md

```diff
@@ -190,4 +187,3 @@ Happy coding!
 [Flutter]: https://docs.flutter.dev/get-started/install
 [git]: https://git-scm.com/downloads
 [Android Studio]: https://developer.android.com/studio
-[Visual Studio Code]: https://code.visualstudio.com/download
```

Comment:
This shouldn't be removed (and the link above restored)

Reply:
Fixed.